src/lib/util/file-upload.ts

```diff
@@ -97,3 +130,18 @@ export async function uploadFilesWithDialog() {
   input.onchange = onManualSourceUpload;
   input.click();
 }
+
+async function getResponseFromModal(
+  currentTableName
+): Promise<DuplicateActions> {
+  duplicateSourceName.set(currentTableName);
```

Comment:
Better to reset the writable to `DuplicateActions.None` when this is called. Or in this method rather than outside.